### PR TITLE
Add Vue.js single file component support

### PR DESCRIPTION
**Pull request by ozanmuyes**
Hi Pete, thank you for awesome plugin.
I work with Vue.js and also use Stylelint for CSS files. But whenever I mix those, the lint support is gone. I stumbled upon [this comment](https://github.com/kungfusheep/SublimeLinter-contrib-stylelint/issues/46#issuecomment-270943648) by @scottsandersdev and tried what he said. It worked (except as he stated, for the top level components) so I thought it's good to have it on the plugin's source code.
Long story short here is the PR for the linter to support Vue.js single file components as well.
Thanks in advance.

### CODE REVIEW

**Comment by kungfusheep** [comment]
Thanks - Merged - sorry for the delay.

**Comment by ozanmuyes** [comment]
Awesome, thanks.
